src/courseContainer.js

```diff
@@ -0,0 +1,74 @@
+                        Take one of Udemy’s range of Python courses and learn how to code
+                        using this incredibly useful language. Its simple syntax and
+                        readability makes Python perfect for Flask, Django, data science,
+                        and machine learning. You’ll learn how to build everything from
+                        games to sites to apps. Choose from a range of courses that will
+                        appeal to...
+                    </p>
+                </div>
+                <button className="btn">Explore  Python</button>
+                <div id="courseCardSection" className="Cards">
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                    <Card courseName="learn python from zero to hero" instructor="ibrahim ramadan"
+                    imagePath="./images//courses/javascript/1.PNG" imageAlt="python course"
+                    price="107" />
+                </div>
```

Comment:
you shouldn't add the cards like this. you can pass the course list as a prop and iterate over them here and pass the json of the course to the Card component as prop. 
something like that 
courses.map((course) => {
        return <Card key={course.id} {...course}></Card>;
    });

Reply:
this was in phase 2 task